src/protobuf-net.Grpc.AspNetCore/ServicesExtensions.cs

```diff
@@ -30,6 +31,21 @@ public static IGrpcServerBuilder AddCodeFirstGrpc(this IServiceCollection servic
             return builder;
         }
 
+        public static IGrpcServerBuilder WithBinderConfiguration(this IGrpcServerBuilder builder, Action<BinderOptions> binderOptions)
+        {
+            var options = new BinderOptions();
+            binderOptions(options);
+            builder.Services.TryAddSingleton(BinderConfiguration.Create(options.MarshallerFactories, options.ServiceBinder));
+            return builder;
+        }
+
+        public sealed class BinderOptions
+        {
+            public IList<MarshallerFactory>? MarshallerFactories { get; set; } = null;
```

Comment:
not sure what this is doing or why it exists; PR doesn't mention it; what is the goal here? how is this different to the ambient marshaller setup that can be injected?

Reply:
This was just a wrapper as it felt a little odd explicitly listing `null` for the marshallers
i.e.
```cs
config => 
{
  config.ServiceBinder = new MyServiceBinder()
}
```
instead of just passing in
```cs
BinderConfiguration.Create(null, new MyServiceBinder)
```